raven/transport/threaded.py

```diff
@@ -64,18 +65,20 @@ def main_thread_terminated(self):
 
             if not self._timed_queue_join(initial_timeout):
                 # if that didn't work, wait a bit longer
-                # NB that size is an approximation, because other threads may
-                # add or remove items
-                size = self._queue.qsize()
-
-                print("Sentry is attempting to send %i pending error messages"
-                      % size)
-                print("Waiting up to %s seconds" % timeout)
-
-                if os.name == 'nt':
-                    print("Press Ctrl-Break to quit")
-                else:
-                    print("Press Ctrl-C to quit")
+
+                if sys.stdout.isatty():
```

Comment:
Hm... as we are printing to stdout, I'd propose to - if you want to test for stdin - to test for both. At least, I also don't want to see those messages when redirecting a console application to a file, maybe by nohup or something.

>>> import sys
>>> sys.stdout.isatty()
True

on my machine.